commands/failover.md

```diff
@@ -0,0 +1,43 @@
+This command will start a coordinated failover between a master and one of its replicas. The failover is not synchronous, instead
+a background task will handle coordinating the failover. It is designed to limit 
+data loss and unavailability of the cluster during the failover. This command is analogous 
```

Comment:
i think we should avoid manual word wrapping, and start every sentence in a new line.
see my recent changes here (aimed to create diffs that are easier to review)
https://github.com/redis/redis-doc/pull/1485

Reply:
Ack, good to know about